packages/mikro-orm/src/index.ts

```diff
@@ -0,0 +1,148 @@
+import { RequestContext, wrap } from "@mikro-orm/core"
+
+import {
+  Account,
+  Session,
+  User as DefaultUserModel,
+  VerificationToken,
+} from "./models"
```

Comment:
```suggestion
import * as defaultModels from "./models"
```

I would use a * import and re-export the models as well to the user, in case they could use it for something else.